.github/workflows/labeled_issues.yml

```diff
@@ -9,6 +9,6 @@ jobs:
     - uses: aareet/move-labeled-or-milestoned-issue@v2.0
       with:
         action-token: "${{ secrets.k8s_github_actions }}"
-        project-url: "https://github.com/orgs/terraform-providers/projects/19"
+        project-url: "https://github.com/terraform-providers/terraform-provider-kubernetes/projects/3"
```

Comment:
This URL is giving me a 404. Is it correct?

Reply:
Yeah, see https://hashicorp.slack.com/archives/GLQJGB94Z/p1591377518302900. I'll close this branch for now, we'll have to make changes again after the repo migration.